nodejs/src/apis/interfaces/common.ts

```diff
@@ -1,4 +1,13 @@
 export interface QueryResults<T> {
   limit: number;
-  data: T[];
+  data?: T[];
```

Comment:
when would data be undefined?

Reply:
Because there were responses for endpoints like:
`{ limit: 10, data: null }`